### ISSUE TITLE
Support for spring boot 3.2.0

### ISSUE THREAD

**Issue by sahassakhare** [Playtika/feign-reactive#663]
Hi team, when can we expect the support for spring boot 3.2.0. the current release version is only working till spring boot 3.1.6 and not for 3.2.0.

**Comment by razvdana** [comment]
I am also currently migrating to spring boot 3.2.1 and I have incompaibility issues. 
Application context loading fails on ReactiveFeignClientFactoryBean.

```
Caused by: java.lang.IllegalArgumentException: Invalid value type for attribute 'factoryBeanObjectType': java.lang.String
```
Is there any workaround until then ?

**Comment by ramzes3333** [comment]
Is this a duplicate to https://github.com/PlaytikaOSS/feign-reactive/issues/651?

**Comment by razvdana** [comment]
> Is this a duplicate to #651?

Indeed, thanks for the link

**Comment by sahassakhare** [comment]
Hi team, any plans to release spring boot 3.2.x compatible version ?

**Comment by Jardo-51** [comment]
I have created a [fork](https://github.com/Jardo-51/feign-reactive) that fixes the Spring Boot 3.2 issue and I published a new version to Maven Central.

Just replace the starter dependency in your `pom.xml` with this, and it should work:
```xml
		<dependency>
			<groupId>com.jardoapps.reactivefeign</groupId>
			<artifactId>feign-reactor-spring-cloud-starter</artifactId>
			<version>4.1.0</version>
			<type>pom</type>
		</dependency>
```
If you need any other modules, just replace the groupId with `com.jardoapps.reactivefeign` and the version with `4.1.0`.

Credit for the fix belongs to @ramzes3333.

**Comment by wdallastella** [comment]
@kptfh are you still supporting this library?

**Comment by kptfh** [comment]
released version 4.1.0 with support of SB3.2
fill free to reopen this ticket in case of any issues
